src/states/Lobby.gd

```diff
@@ -38,25 +38,34 @@ onready var start_button := $"%StartButton" as Button
 var chosen_colors := {}
 # Dictionary from player_id to row in the table.
 var player_id_to_row := {}
+# Dictionary from player_id to ping, in ms.
+var player_id_to_ping := {}
```

Comment:
So here was my thought process:

1. `Multiplayer.player_latency` is only valid for the server, since it's the one that actually does the pinging.
2. We need to sync this to all the peers.
3. Having the peers update their own `player_latency` value seems a bit sus, especially if we ever want to do actual peer-to-peer stuff with latency there

Solution: Have a variable that's a copy of `player_latency` for the server, and a copy of the server's `player_latency` for all the clients.

Reply:
Gotcha. Would probably be better to just rpc over the `Multiplayer.player_latency` from servers to clients so there's only one copy but no big deal